File: app/src/main/java/com/example/yelpbusinessexample/data/model/BusinessResponse.java
package com.example.yelpbusinessexample.data.model;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class BusinessResponse{

	@SerializedName("total")
	private int total;

	@SerializedName("region")
	private Region region;

	@SerializedName("businesses")
	private List<Businesses> businesses;

	public void setTotal(int total){
		this.total = total;
	}

	public int getTotal(){
		return total;
	}

	public void setRegion(Region region){
		this.region = region;
	}

	public Region getRegion(){
		return region;
	}

	public void setBusinesses(List<Businesses> businesses){
		this.businesses = businesses;
	}

	public List<Businesses> getBusinesses(){
		return businesses;
	}
}

File: app/src/main/java/com/example/yelpbusinessexample/data/model/BusinessResponse.kt
package com.example.yelpbusinessexample.data.model

import com.google.gson.annotations.SerializedName
import com.example.yelpbusinessexample.data.model.Businesses

data class BusinessResponse(
    @SerializedName("total") var total : Int,

    @SerializedName("region") var region: Region? = null,

    @SerializedName("businesses") var businesses: List<Businesses>? = null
)